Dgreen15/update v3 mapping docs (#27135)
DataDog/documentation: content/en/service_catalog/service_definitions/v3-0.md

### v3.0 API endpoints
To view v3.0 API endpoints, see the latest [API documentation][5].

### Migrating to v3.0

> v3.0 supports the same methods of creating metadata as previous version including Github, API, Terraform, Backstage, ServiceNow, and the UI. However, there are new [API endpoints][5] and a new [Terraform module][6] for v3.0.

Reviewer: ```suggestion
v3.0 supports the same methods of creating metadata as previous versions, including Github, API, Terraform, Backstage, ServiceNow, and the UI. However, there are new [API endpoints][5] and a new [Terraform module][6] for v3.0.
```

You can see the [full schema definitions on Github][1]. V3.0 contains the following changes from v2.2:
- `schema_version` is now `api_version`

> - `kind` field is new to delineate the type of component: service, queue, datastore, system, or API

Reviewer: ```suggestion
- `kind` field is new and defines the type of component: service, queue, datastore, system, or API
```
